app/views/container_service/_main.html.haml

```diff
@@ -1,8 +1,15 @@
 = render :partial => "layouts/flash_msg"
 .row
   .col-md-12.col-lg-6
-    = render :partial => "shared/summary/textual", :locals => { :title => _("Properties"), :items => textual_group_properties }
-    = render :partial => "shared/summary/textual_multilabel", :locals => { :title => _("Port Configurations"), :items => textual_group_port_configs }
+    = render :partial => "shared/summary/textual", :locals => {:title => _("Properties"),
```

Comment:
> I understand that, but I'm wondering - has Rubocop alerted on those lines even though you were not supposed to change them as part of this patch? (those are not label/selector lines)

Yes, but not as part of the miq-bot, I ran rubocop locally.
 It seems there's a problem with miq-bot alerts  (as you can see above, it warns about parts of the code unrelated to the patch) and I understood from @zeari it's OK we cosmetically fix parts as part of our patches.